packages/graphql/tests/integration/types/date.int.test.ts

```diff
@@ -189,7 +189,7 @@ describe("Date", () => {
                 mutation {
                     ${
                         Movie.operations.update
-                    }(where: { id_EQ: "${id}"}, update: { date_SET: "${date.toISOString()}" }) {
+                    }(where: { id_EQ: "${id}"}, update: { date: {set: "${date.toISOString()}" } }) {
```

Comment:
same reason as above:
```suggestion
                    }(where: { id: { equals: "${id}" } }, update: { date: {set: "${date.toISOString()}" } }) {
```

Reply:
I'd prefer to keep this PR without those changs to avoid conflicts, particularly after the discussion on changing the names for the filters (`equals` vs `eq`)

Reply:
Cool, no problem, we can change it later on when we migrate all the tests!